src/crate/theme/rtd/conf/__init__.py

```diff
@@ -99,7 +99,8 @@
 # Configure intersphinx mapping
 intersphinx_mapping = {
 
-    # CrateDB Core
+    # CrateDB General
+    'guide': ('https://cratedb.com/docs/guide/', None),
     'crate-reference': ('https://cratedb.com/docs/crate/reference/en/latest/', None),
     'crate-tutorials': ('https://cratedb.com/docs/crate/tutorials/en/latest/', None),
     'crate-howtos': ('https://cratedb.com/docs/crate/howtos/en/latest/', None),
```

Comment:
This change only adds a new project, and does not decommission the previous intersphinx link targets, so it does not break downstream projects.